CHANGES.md

```diff
@@ -1,3 +1,22 @@
+## edge-19.7.4
+
+* CLI
+  * Made the `linkerd routes` command traffic-split aware; consider resources
```

Comment:
is "consider resources" referring to the behavior of the command? I might say "it considers resources" (as long as that's correct).

Reply:
Good catch :+1:

Reply:
What about just eliding that part of the sentence? Feels overly implementation-detail-y for release notes.